src/internal/operators/endWith.ts

```diff
@@ -46,7 +46,7 @@ export function endWith<T, A extends unknown[] = T[]>(...values: A): OperatorFun
  *   endWith('interval ended by click'),
  * )
  * .subscribe(
- *   x = console.log(x);
+ *   x => console.log(x)
  * )
```

Comment:
`;` could be added here since almost all other examples use `;`. This file uses it as well. Maybe the whole `subscribe` call could be written in a single line...

Reply:
`;` here is a syntax error. As you mentioned, it will be better to make it a single line, thx!